File: programacao/c/respostas/selecao/39.c
#include <stdio.h>

int main(void) {
    float a, b, c;

    printf("Digite a, b e c: ");
    scanf("%f %f %f", &a, &b, &c);

    if (a + b > c && a + c > b && b + c > a) {
        printf("Formam triangulo\n");
        if (a == b && b == c) {
            printf("Equilatero\n");
        } else if (a == b || b == c || a == c) {
            printf("Isosceles\n");
        } else {
            printf("Escaleno\n");
        }
    } else {
        printf("Nao formam triangulo\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/39.py
a = float(input("Digite a: "))
b = float(input("Digite b: "))
c = float(input("Digite c: "))

if a + b > c and a + c > b and b + c > a:
    print("Formam triangulo")
    if a == b and b == c:
        print("Equilatero")
    elif a == b or b == c or a == c:
        print("Isosceles")
    else:
        print("Escaleno")
else:
    print("Nao formam triangulo")
